src/main/java/local/alfonso/orders/model/Customers.java

```diff
@@ -0,0 +1,190 @@
+package local.alfonso.orders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "customers")
+public class Customers
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long custcode;
+
+    @Column(nullable = false)
+    private String custname;
+
+    private String custcity;
+    private String workingarea;
+    private String custcountry;
+    private String grade;
+    private double openingamt;
+    private double receiveamt;
+    private double paymentamt;
+    private double outstandingamt;
+    private String phone;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "agentcode",
+            nullable = false)
+    @JsonIgnoreProperties({"customers", "hibernateLazyInitializer"})
```

Comment:
Since you're using `fetchType.LAZY` here, make sure to ignore the property `hibernateLazyInitializer` as well to prevent its information from printing unnecessarily.